index.html

```diff
@@ -50,11 +50,15 @@ <h4>Homoglyphs</h4>
       <ul>
       <li><a href="https://еxample.test">URL uses Cyrillic е instead of e</a></li>
       <li><a href="https://еxӑmple.test">URL uses Cyrillic ӑ instead of a</a></li>
-      <li><a href="https://a.example.test">URL uses Cyrillic a in its own label</a></li>
+      <li><a href="https://а.example.test">URL uses Cyrillic a in its own label</a></li>
       <li><a href="https://ӑ.example.test">URL uses Cyrillic ӑ in its own label</a></li>
       <li><a href="https://examp1e.test">URL uses number one instead of l</a></li>
       <li><a href="https://bankofthevvest.com">URL uses "vv" instead of "w"</a></li>
       <li><a href="https://go0gle.com">URL uses "0" instead of "o"</a></li>
+      <li><a href="https://aO12.com">URL with mainly numbers uses "O" instead of "0"</a></li>
+      <li><a href="https://a0l2.com">URL with mainly numbers uses "l" instead of "1"</a></li>
+      <li><a href="https://а012.com">URL with mainly numbers uses Cyrillic a instead of a</a></li>
```

Comment:
I'd prefer something like "That uses a single Cyrillic letter and numbers" since it's not clear that 'a' would be the original (like in the example.test with Cyrillic a case). Same in the greek case below